src/com/lftechnology/java/training/niraj/ems/domains/Pojo.java

```diff
@@ -0,0 +1,39 @@
+package com.lftechnology.java.training.niraj.ems.domains;
+
+import java.util.List;
+import java.util.Map;
+
+public abstract class Pojo {
+    public String table;
+    public String primaryKey;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make primaryKey a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)